test/e2e_env/gatewayapi/conformance_test.go

```diff
@@ -106,7 +106,7 @@ func TestConformance(t *testing.T) {
 		RestConfig:           clientConfig,
 		Clientset:            clientset,
 		GatewayClassName:     "kuma",
-		CleanupBaseResources: true,
+		CleanupBaseResources: !Config.Debug,
```

Comment:
When the resources are cleaned up, we can't collect logs from these containers. We disable cleaning up when debug is enabled so that the logs/state are dumped to files.